step3/app.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+PORT = 3000;
```

Comment:
```suggestion
const PORT = 3000;
```
혹시 모르니 이렇게 `const`로 정의해주세요!